ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -327,4 +354,45 @@ private static boolean bug1733118Affected(String namespace, String docType, Obje
             .matches("0[.]([0-9]|1[0-6])[.].*"); // < 0.17
   }
 
+  // See bug 1751753 for explanation of context.
+  private static void processForBug1751753(ObjectNode json) {
+    // Sanitize keys in the SEARCH_COUNTS histogram.
+    JsonNode searchCounts = json.path("payload").path("keyedHistograms").path("SEARCH_COUNTS");
+    processKeysForBug1751753(searchCounts);
+
+    // Sanitize keys in browser.search.content.* keyed scalars.
+    json.path("payload").path("processes").path("parent").path("keyedScalars") //
+        .fields().forEachRemaining(entry -> {
+          if (entry.getKey().startsWith("browser.search.content.")) {
+            processKeysForBug1751753(entry.getValue());
+          }
+        });
+  }
+
+  private static void processKeysForBug1751753(JsonNode inputNode) {
+    if (!inputNode.isObject()) {
+      // If this isn't an object, we have nothing to do here.
+      return;
+    }
+    ObjectNode searchNode = (ObjectNode) inputNode;
+    Lists.newArrayList(searchNode.fieldNames()).forEach(name -> {
```

Comment:
It seems odd to me to rely on this ancillary `Lists` class from the Google API client.  I'd suggest importing `org.apache.commons.collections.IteratorUtils` and using that:
```suggestion
    IteratorUtils.toList(searchNode.fieldNames()).forEach(name -> {
```